ios/Runner/LocaleMeasurementHandler.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  LocaleMeasurementHandle.swift
+//  Runner
+//
+//  Created by Le Phuoc on 31/05/2023.
+//
+
+import Foundation
+
+class LocaleMeasurementHandler {
+
+    static let shared = LocaleMeasurementHandler()
+    
+    func getLocale(call: FlutterMethodCall, result: @escaping FlutterResult) {
+        var locale: String?
+        if #available(iOS 16.0, *) {
+            locale = Locale.current.measurementSystem.identifier
+        } else {
+            locale = Locale.current.regionCode
```

Comment:
since it is the same logic will the fallback code in flutter, I think we can fallback to return empty or a default value when ios < 16.

Just make the function clear logic to fetch `measurementSystem` settings and keep it doesn't mess with the `regionCode`, default if not available.